setup.py

```diff
@@ -1,16 +1,31 @@
 from setuptools import setup, find_packages
 
-required_packages = [
+install_requires = [
     "graphql-core>=2.3,<3",
     "flask>=0.7.0",
     "graphql-server-core>=1.1,<2",
 ]
 
+tests_requires = [
+    'pytest>=2.7.2',
+    'pytest-cov==2.8.1',
+    'pytest-flask>=0.10.0',
+    'graphql-core>=2.1,<3',
+    'graphql-server-core>=1.1,<2',
+    'Flask>=0.10.0',
```

Comment:
Why specify a different version of Flask here? Also `graphql-core` and `graphql-server-core` are already defined in the `install_requires` list so they don't need to be in the `tests_requires` as well.

Reply:
Oops! Probably copy & paste error while moving the requirements from `tox` file.